### PR TITLE
chore: adds shauna to README and updates documentation to reflect usage of all-contributors CLI

### PR DESCRIPTION
**Pull request by Shkeating**
# Summary

Re-does adding myself to the contributors file, and improves documentation to prevent similar mistakes from future contributors. In this branch I did three things:

- removes my manually added code from the README file to prevent issues with the CLI tool moving forward
- adds myself using the CLI tool
- updates the `For Trussels` file to instruct new people to use the tool so they don't make the same mistake I did 😅

## Related Issues or PRs

https://github.com/trussworks/react-uswds/pull/2193 - where I added myself manually 

## How To Test

- check if I show up in the README preview
- Review the text I wrote in the `for-trussels.md` file. open to suggestions on how to word it better

### CODE REVIEW

File: docs/for_trussels.md [hunk 1]
```diff
@@ -18,8 +19,8 @@ First of all, we’re so excited you want to be an active maintainer on this pro
 - [ ] _For Maintainers:_ Add yourself to main project README "Active Maintainers" list.
 - [ ] _For Maintainers:_ Ping in #react-uswds to request access to the npm org so you can publish new releases
 - [ ] _For Maintainers:_ Ping in #react-uswds to request admin privileges to the [Happo Account](https://happo.io/)
-- [ ] _Optional:_ Add yourself to the `contributors` list in the [contributors file](../all-contributorsrc.json),
-- [ ] _Optional: Add yourself to [CODEOWNERS](../CODEOWNERS) if you'd like to automatically be requested to review PRs.
+- [ ] _Optional:_ Add yourself to the `contributors` list in the the [the README](../README.md), using the command `yarn contributors:add [your github username] code`
```
Comment (rogeruiz) [inline docs/for_trussels.md:22]:
```suggestion
- [ ] _Optional:_ Add yourself to the `contributors` list in the [the README](../README.md), using the command `yarn contributors:add [your github username] code`
```

File: docs/for_trussels.md [hunk 2]
```diff
@@ -18,8 +19,8 @@ First of all, we’re so excited you want to be an active maintainer on this pro
...
+- [ ] \_Optional: Add yourself to [CODEOWNERS](../CODEOWNERS) if you'd like to automatically be requested to review PRs.
```
Comment (rogeruiz) [inline docs/for_trussels.md:23]:
```suggestion
- [ ] _Optional_: Add yourself to [CODEOWNERS](../CODEOWNERS) if you'd like to automatically be requested to review PRs.
```

**Review by rogeruiz — CHANGES_REQUESTED** [review]
❌ This PR needs changes ...

Thanks @Shkeating for doing this work. I left some suggestions that need to be addressed and can be done via the GitHub UI if you'd like. But otherwise, this looks good and better than #2193

**Review by gidjin — APPROVED** [review]
LGTM

**Pull request merged**